core/openseadragon/openseadragon.js

```diff
@@ -10172,6 +10172,7 @@ function updateOnce( viewer ) {
          * @property {OpenSeadragon.Viewer} eventSource - A reference to the Viewer which raised this event.
          * @property {?Object} userData - Arbitrary subscriber-defined object.
          */
+        unenhance();
```

Comment:
Openseadragon is used more broadly than just viewer, so this breaks all other apps.

It may be more useful to use an event handler? The [openseadragon docs](https://openseadragon.github.io/docs/OpenSeadragon.EventSource.html) may be helpful for that.